Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1167,6 +1176,41 @@ WarpXOpenPMDPlot::GetMeshCompNames (int meshLevel,
 
     field_name += std::string("_lvl").append(std::to_string(meshLevel));
 }
+/*
+ *
+ *
+ */
+void
+GetFieldModeIndices(const std::string& varname,
```

Comment:
Our little AMReX convention:
```suggestion
GetFieldModeIndices (const std::string& varname,
```